src/wallet/wallet.cpp

```diff
@@ -1459,7 +1459,16 @@ set<uint256> CWallet::GetConflicts(const uint256& txid) const
         }
     }
 
-    // TODO ORCHARD; see #5593
+    for (const uint256 nullifier: wtx.GetOrchardBundle().GetNullifiers()) {
+        if (mapTxOrchardNullifiers.count(nullifier) <= 1) {
+            continue;  // No conflict if zero or one spends
+        }
+        range_o = mapTxOrchardNullifiers.equal_range(nullifier);
+        for (TxNullifiers::const_iterator it = range_o.first; it != range_o.second; ++it) {
+            // TODO: Take into account transaction expiry; see #5585
+            result.insert(it->second);
+        }
+    }
```

Comment:
`mapTxOrchardNullifiers` is newly added and is not populated anywhere. Instead, get this information from the `potential_spends` field of the Orchard sub-wallet.